website_sale_attribute_filter_price/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - Alexandre Díaz
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Wesbite Sale Attribute Filter Price",
```

Comment:
Ooops! I forgot this...
```suggestion
    "name": "Website Sale Attribute Filter Price",
```